src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -482,10 +526,13 @@ const WorkflowView = _.flow(
         savedConfig: config, modifiedConfig,
         currentSnapRedacted: isRedacted, savedSnapRedacted: isRedacted,
         entityMetadata,
+        availableSnapshots: snapshots,
+        selectedEntitySource,
```

Comment:
Would a more appropriate piece of state be to make this a boolean called `snapshotSourceSelected` rather than doing string comps everywhere?